### DISCUSSION TITLE
How to remove the extra space around inline formula?

### DISCUSSION THREAD

**Discussion by QiangF** [kjambunathan/org-mode-ox-odt#275] [Q&A] [answered]
Export the following snippet:

    文字 $%alpha_{K}$ 文字，Text $%alpha_{K}$ Text

The space between text and formula is too large, is there a way to remove it?

**Comment by kjambunathan** [comment, marked as answer]
> Export the following snippet:
> 
> ```
> 文字 $%alpha_{K}$ 文字，Text $%alpha_{K}$ Text
> ```

**_Downloadable zip file_**

This zip file that contains all the artefacts [discussion-275-custom-typesetting-of-inline-math-formula.zip](https://github.com/user-attachments/files/16787883/discussion-275-custom-typesetting-of-inline-math-formula.zip)

----------------



To get this

**_Desired Output_**

![03-How the formula is typeset after Customization](https://github.com/user-attachments/assets/05ba76a6-64be-455e-b23e-03c1fafee061)

do this 

**_How to change the Font Size of the Formula_**

![01-How to Configur Formula Size](https://github.com/user-attachments/assets/273cbbf0-beab-4afa-8273-74395d6f6441)

and this

**_How to change the borders around the Formula_**

![02-How to change formula borders](https://github.com/user-attachments/assets/2c0f9fb2-106d-4e94-bf74-fa92827597c7)

and then export this `org' snippet

**_Org Snippet_**

```
#+options: ':nil *:t -:t ::t <:t H:3 \n:nil ^:t arch:headline
#+options: author:t broken-links:mark c:nil creator:nil
#+options: d:(not "LOGBOOK") date:t e:t email:nil expand-links:t f:t
#+options: inline:t num:t p:nil pri:nil prop:nil stat:t tags:t
#+options: tasks:t tex:t timestamp:t title:t toc:t todo:t |:t
#+title: formula
#+date: <2024-08-28 Wed>
#+author: Jambunathan K
#+email: kjambunathan@gmail.com
#+language: en
#+select_tags: export
#+exclude_tags: noexport
#+creator: Emacs 31.0.50 (Org mode 9.7.7)
#+cite_export:

#+odt_preferred_output_format:
#+odt_styles_file:
#+odt_extra_images:
#+odt_extra_styles:
#+odt_extra_automatic_styles:
#+odt_master_styles:
#+odt_content_template_file:
#+odt_automatic_styles:

文字 $\alpha_{K}$ 文字，Text $\alpha_{K}$ Text
```

followed by

**_Open the exported `odt` file in `LO` and save it; this ensures that `settings.xml ` is persisted in the final `ODT` file_**

![04-Resave the exported file with LO so that formula `settings xml' is saved to ODT file](https://github.com/user-attachments/assets/507d14e7-dfcb-493b-8657-f0cd475e484d)
